data/data/aws/vpc/sg-master.tf

```diff
@@ -216,6 +216,16 @@ resource "aws_security_group_rule" "master_ingress_etcd" {
   self      = true
 }
 
+resource "aws_security_group_rule" "master_ingress_etcd_metrics" {
+  type              = "ingress"
+  security_group_id = "${aws_security_group.master.id}"
+
+  protocol  = "tcp"
+  from_port = 23790
```

Comment:
can you get approval to use that port https://github.com/openshift/origin/pull/21520